zqd/ingest/log.go

```diff
@@ -158,7 +154,8 @@ func (p *LogOp) Status() <-chan string {
 	return p.warningCh
 }
 
+// Error indicates what if any error occurred during import, after the
+// Status channel is closed.
 func (p *LogOp) Error() error {
-	p.wg.Wait()
```

Comment:
Correct me if I'm wrong but doesn't this make this call no longer thread safe?

Reply:
@mattnibs: It's safe after the Status channel is closed (but not before).